proxy-init/iptables/iptables.go

```diff
@@ -241,6 +244,20 @@ func makeJumpFromChainToAnotherForAllProtocols(chainName string, targetChain str
 		"--comment", formatComment(comment))
 }
 
+func makeRedirectChainForOutgoingTraffic(chainName string, redirectChainName string, comment string) *exec.Cmd {
+	return exec.Command("iptables",
+		"-t", "nat",
+		"-A", chainName,
+		"-m", "owner",
+		"--uid-owner", "2102",
```

Comment:
This user ID is configurable when running `linkerd inject --proxy-uid 1234 -` -- unfortunately we can't hardcode it here. But the configured UID is passed into this process and stored in the `ProxyUid` field of the `FirewallConfiguration` struct in this file. So you should be able to use that. See my suggestion above.